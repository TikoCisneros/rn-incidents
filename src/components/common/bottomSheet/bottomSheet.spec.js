import React from 'react';
import { shallow } from 'enzyme';
import BottomSheet from './bottomSheet';

describe('BottomSheet component tests', () => {
  test('Should render correctly', () => {
    const snapshot = shallow(<BottomSheet height={200} />);
    expect(snapshot).toMatchSnapshot();
  });
});
