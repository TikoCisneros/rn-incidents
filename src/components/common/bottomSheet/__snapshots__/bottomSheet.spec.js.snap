// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomSheet component tests Should render correctly 1`] = `
<RBSheet
  animationType="slide"
  closeDuration={200}
  closeOnDragDown={true}
  closeOnPressBack={true}
  closeOnPressMask={false}
  customStyles={
    Object {
      "container": Object {
        "borderTopLeftRadius": 10,
        "borderTopRightRadius": 10,
      },
      "draggableIcon": Object {
        "backgroundColor": "#804ba1",
      },
      "wrapper": Object {
        "backgroundColor": "rgba(46, 49, 49, 0.3)",
      },
    }
  }
  dragFromTopOnly={false}
  height={200}
  keyboardAvoidingViewEnabled={true}
  minClosingHeight={0}
  onClose={null}
  onOpen={null}
  openDuration={300}
>
  <View />
</RBSheet>
`;
